.clang-format

```diff
@@ -1,16 +1,26 @@
 ---
-BasedOnStyle: Google
-Language: Cpp
-Standard: Cpp03
+# Indentation for all files
+ColumnLimit: 120
+IndentWidth: 4
 TabWidth: 4
 UseTab: Never
-ColumnLimit: 120
-NamespaceIndentation: All
+
+# C++ Options
+Language: Cpp
+Standard: c++17
+
+BasedOnStyle: Google
+
+IncludeBlocks: Preserve
 
 AccessModifierOffset: -4
```

Comment:
This value is connected to the above Indent, per default it's `-1`. Do we really need this diversion from default?